
float32 x		# x of position
float32 y		# y of position
float32 w		# width
float32 h		# height
